### PR TITLE
feat(volo-cli):Support auto generate default impl code in lib.rs in volo-cli init command

### PR DESCRIPTION
**Pull request by my-vegetable-has-exploded**
## Motivation

#5 

## Solution

Because the service may  use the variables of the imported package, so we need to use global path other than cur_related_path in codegen_service_method.

- Implement codegen_service_method_with_global_path
- Build InitBuilder which call pilota-build to get service_name and methods

I did some test by using volo-cli command, but how can i add some ut?  pilota's unit tests seem to be few, I don't know how to write ut for it.

### CODE REVIEW

File: volo-cli/src/templates/grpc/src/lib_rs [hunk 1]
```diff
@@ -1,6 +1,11 @@
 #![feature(impl_trait_in_assoc_type)]
 
+use tokio::sync::mpsc;
+use tokio_stream::{{wrappers::ReceiverStream, StreamExt}};
```
Comment (Millione) [inline volo-cli/src/templates/grpc/src/lib_rs:4]: How about re-export these crates in volo-grpc and use global path in volo-build [here](https://github.com/cloudwego/volo/pull/207/files#diff-0f7759de6a3422548d84e834c44007de0a95c9108b784757c629a35cb0c89b93R77)?

Comment (my-vegetable-has-exploded) [reply volo-cli/src/templates/grpc/src/lib_rs:4]: This is a good way that I did not think of before.

Comment (my-vegetable-has-exploded) [reply volo-cli/src/templates/grpc/src/lib_rs:4]: > How about re-export these crates in volo-grpc and use global path in volo-build [here](https://github.com/cloudwego/volo/pull/207/files#diff-0f7759de6a3422548d84e834c44007de0a95c9108b784757c629a35cb0c89b93R77)?

Do I need to re-export these crates entirely?  Re-exporting these crates entirely seems to involve a lot of changes.

Comment (Millione) [reply volo-cli/src/templates/grpc/src/lib_rs:4]: You can re-export like [this](https://github.com/cloudwego/volo/blob/main/volo-grpc/src/codegen.rs)

Comment (my-vegetable-has-exploded) [reply volo-cli/src/templates/grpc/src/lib_rs]: get it

File: volo-grpc/src/lib.rs [hunk 2]
```diff
@@ -28,3 +28,5 @@ pub use message::{RecvEntryMessage, SendEntryMessage};
 pub use request::{IntoRequest, IntoStreamingRequest, Request};
 pub use response::Response;
 pub use status::{Code, Status};
+pub use tokio::sync::mpsc;
```
Comment (Millione) [inline volo-grpc/src/lib.rs:31]: put it in codegen.rs?

**Comment by Millione** [comment]
LGTM

**Comment by my-vegetable-has-exploded** [comment]
This pr is dependent on pilota‘s modifications. After releasing next pilota crate , I think that the ci will pass.

**Comment by Millione** [comment]
> This pr is dependent on pilota‘s modifications. After releasing next pilota crate , I think that the ci will pass.

Just change the crates.io pilota dependencies to git in Cargo.toml.

**Review by Millione — DISMISSED** [review]

**Comment by my-vegetable-has-exploded** [comment]
```rust
❯ cargo fmt -- --check
Diff in /home/wy/rust/volo/volo-thrift/src/codec/default/thrift.rs at line 172:
             // not enough bytes to detect, so return error
             return Err(pilota::thrift::error::DecodeError::new(
                 DecodeErrorKind::BadVersion,
-                    "not enough bytes to detect protocol in thrift codec",
-                ),
-            );
+                "not enough bytes to detect protocol in thrift codec",
+            ));
         };
 
         // detect protocol
```
It happens with lastest nightly cargo. Should I fmt it in this pr or  in a new pr？

**Review by Millione — APPROVED** [review]

**Review by PureWhiteWu — APPROVED** [review]

**Comment by PureWhiteWu** [comment]
Thanks for your contribution!

**Comment by my-vegetable-has-exploded** [comment]
thanks.

**Pull request merged**